packages/main/src/ToolbarButtonTemplate.tsx

```diff
@@ -4,11 +4,11 @@ import Button from "./Button.js";
 export default function ToolbarButtonTemplate(this: ToolbarButton) {
 	return (
 		<Button
-			class="ui5-tb-button ui5-tb-item"
+			class={this.class}
 			id={this.id}
 			style={{
-				width: this.width,
-				display: this.hidden ? "none" : "inline-block",
+				width: this.width || "100%",
+				display: this.hidden ? "none" : "block",
```

Comment:
The line `display: this.hidden ? "none" : "block",` must no longer be needed as the hidden attribute will affect the physical item. We also have a private property `hidden` https://github.com/SAP/ui5-webcomponents/pull/10915#issuecomment-2698068566 that can also be removed.